src/output/plugins/httpd/HttpdClient.cxx

```diff
@@ -7,6 +7,7 @@
 #include "util/AllocatedString.hxx"
 #include "Page.hxx"
 #include "IcyMetaDataServer.hxx"
+#include "lib/crypto/Base64.hxx"
```

Comment:
This header is only available if the macro `HAVE_BASE64` is defined (don't forget to `#include "config.h"`), and then you need to add a Meson library dependency on `crypto_base64_dep`.

Reply:
Done.

Reply:
No, you havn't done that.

Reply:
Ooops.